msi/GitProduct.wxs

```diff
@@ -12,7 +12,7 @@
            UpgradeCode='abbb3e1d-fea0-4b84-9585-b4dc13cbc1b4'>
     <Package InstallScope='perMachine' Compressed='yes' />
 
-    <MajorUpgrade DowngradeErrorMessage='A newer version of [ProductName] is already installed.' />
+    <MajorUpgrade DowngradeErrorMessage='A newer version of [ProductName] is already installed.' Schedule="afterInstallInitialize" />
```

Comment:
I have to admit that I would have been *completely* lost what this diff does without this very pleasant commit message. Well done.